packages/mdc-shape/mdc-shape.scss

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+$mdc-shape-shadow-double-padding: 2 * $mdc-shape-shadow-padding;
```

Comment:
I'm confused why we're doing this here yet we have mdc-shape-shadow-negative-padding in variables.scss. For that matter, maybe it'd be easier to understand if we simply set all of top/left/bottom/right to the same thing, instead of requiring this math?

I'm also not sure I understand what the padding on each side of the canvas is useful for?

Reply:
I think I was struggling with how to use a Sass variable within a calc function....fixed it now

The padding is to make sure the canvas is large enough to cover the highest elevation (24) with the largest blur (56px). This way there will always be enough room on the canvas to draw the shadow, regardless of what elevation a shape is at.